server/fleet/packs.go

```diff
@@ -169,13 +169,6 @@ type PackSpecQuery struct {
 	Denylist    *bool   `json:"denylist,omitempty"`
 }
 
-// PackTarget targets a pack to a host, label, or team.
-type PackTarget struct {
-	ID     uint `db:"id" json:"-"`
-	PackID uint `db:"pack_id" json:"-"`
-	Target
-}
-
```

Comment:
Not sure I understand how this is related to the fix.

Reply:
Not related, just found that the type is not used at all.